### DISCUSSION TITLE
Adding/removing UI languages

### DISCUSSION THREAD

**Discussion by adanielyan** [tolgee/tolgee-platform#1589] [Q&A]
Hi, how can we add/remove the list of languages in the UI language selector? 
<img width="323" alt="image" src="https://user-images.githubusercontent.com/861777/221764431-a0fcdf22-b316-453c-bdb5-82bd7eff0140.png">
I couldn't find this anything in the documentation about it.

**Comment by JanCizmar** [comment]
Tolgee is currently translated only to these languages. So you can't add or remove languages to this select. However, you can help us with translating by joining our slack community: https://tolg.ee/slack
